apps/masterbots.ai/components/icons/infoIcon.tsx

```
@@ -0,0 +1,20 @@
+const InfoIcon = (props: React.ComponentPropsWithoutRef<'svg'>) => (
+  <svg width="18" height="18" viewBox="0 0 18 18" fill="none" {...props}>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)